packages/hash/README.md

```diff
@@ -10,7 +10,9 @@
 
 HASH is an open-source, data-centric, all-in-one workspace. HASH combines a rich frontend editor with a powerful entity graph that makes it easy to capture and work with structured data. HASH is built atop the open [Block Protocol](https://github.com/blockprotocol/blockprotocol) allowing users to easily add new block types and functionality to their workspaces.
 
-**This app is not yet ready for production use.** For now it is intended to be used as a test-harness for developers building Block Protocol compliant blocks.
+**This app is not yet ready for production use.** For now it is intended to be used as a [test-harness for developers building Block Protocol-compliant blocks](#integration-with-the-block-protocol). It is currently not secure, not optimised, and missing key features.
+
+We will be developing HASH into a production-grade application which can be self-hosted. The current design and architecture, while not fully realised, paves the way for further features, scale, and performance. You can read about the long-term vision for HASH [here](https://hash.ai/platform/hash).
```

Comment:
American/intl English plz (with a 'z')

Reply:
Oops. I can't spell behaviour any more - these others will come too, in time.